src/wp-includes/media.php

```diff
@@ -1866,6 +1869,79 @@ function wp_filter_content_tags( $content, $context = null ) {
 	return $content;
 }
 
+/**
+ * Use alternate mime type images in the content output when available.
+ *
+ * @since 6.0.0
+ *
+ * @param string $image         The HTML `img` tag where the attribute should be added.
+ * @param int    $attachment_id The attachment ID.
+ * @return string Converted `img` tag with `loading` attribute added.
+ */
+function wp_image_use_alternate_mime_types( $image, $attachment_id ) {
+	$metadata = wp_get_attachment_metadata( $attachment_id );
+	if ( empty( $metadata['file'] ) ) {
+		return $image;
+	}
+
+	$target_mimes = array( 'image/webp', 'image/jpeg' );
```

Comment:
This is pending the decision in the plugin on which order to use here, but definitely this is okay for now.

Reply:
👍🏼  yep